File: edTechSpark_frontend_v18/edTechSpark_frontend_v18/edtechspark/src/app/models/order.ts
import { OrderItem } from "./orderItem";

export class Order {
    id: string | any;
    userId: number | undefined;
    paymentId: string | any;
    currency: string | any;
    total: number | undefined;
    tax: number | undefined;
    grandTotal: number | undefined;
    billingAddress: string | any;
    createdDate: string | any;
    items: OrderItem[] | any;
}

File: edTechSpark_backend_v18/edTechSpark_backend/edTech.DomainModels/Entities/Order.cs

using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace edTech.DomainModels.Entities
{
    public class Order
    {
        public Order()
        {
            Items = new HashSet<OrderItem>();
        }
        public string Id { get; set; }
        public int UserId { get; set; }
        public string PaymentId { get; set; }
        public DateTime CreatedDate { get; set; }
        public virtual ICollection<OrderItem> Items { get; set; }
    }
}
